salt/profiles/init.sls

```diff
@@ -55,6 +65,7 @@ profiles-uwsgi-service:
         - require:
             - profiles-install
             - profiles-app-config
+            - profiles-clients-config
             - profiles-uwsgi-config
```

Comment:
We probably want to make these 3 requirements for `profiles-install`, as it might do something that requires configuration.

Reply:
(Actually, not `profiles-uwsgi-config`.)

Reply:
(Actually, just `profiles-app-config`.)